janus_core/cli/md.py

```diff
@@ -289,7 +291,9 @@ def md(
         files. Default is {}.
     post_process_kwargs
         Kwargs to pass to post-processing.
-    seed
+    correlation_kwargs : Optional[CorrelationKwargs]
+        Kwrag to pass for on-the-fly correlations.
```

Comment:
```suggestion
        Keyword arguments to pass for on-the-fly correlations.
```
+ default (also post_process_kwargs)